### PR TITLE
Use minios disk package for cross-platform disk stats

### PR DESCRIPTION
**Pull request by sanderpick**
Textile's [powergate](https://github.com/textileio/powergate) has a [Lotus dependency](https://github.com/textileio/powergate/blob/master/go.mod#L15). Powergate is leveraged by [the hub](https://github.com/textileio/textile), which has Linux and Darwin CLI builds. [This syscall](https://github.com/filecoin-project/sector-storage/blob/master/stores/interface.go#L42) prevents us from releasing a Window build:

```
1 errors occurred:
--> windows/amd64 error: exit status 2
Stderr: # github.com/filecoin-project/sector-storage/stores
../../../go/pkg/mod/github.com/filecoin-project/sector-storage@v0.0.0-20200605192746-4b9317d1f08f/stores/interface.go:41:11: undefined: syscall.Statfs_t
../../../go/pkg/mod/github.com/filecoin-project/sector-storage@v0.0.0-20200605192746-4b9317d1f08f/stores/interface.go:42:12: undefined: syscall.Statfs
``` 

This PR just replaces `syscall.Statfs` with `disk.GetInfo` from https://github.com/minio/minio/tree/release/pkg/disk, which makes a library depending on Lotus compile on Windows.

### CODE REVIEW

**Comment by jsign** [comment]
Adding some extra info. Down the path of some dependency imports, the previous step to jump to this repo is the Lotus API using types for its API definitions: see [here](https://github.com/filecoin-project/lotus/blob/857b5962da0100f6e671e814aa44e63b4e7dd5b5/api/apistruct/struct.go#L16).

**Comment by sanderpick** [comment]
Some other lighter packages that I found:
- [go-disk-usage](https://github.com/ricochet2200/go-disk-usage)
- [disk](https://github.com/StalkR/goircbot/tree/master/lib/disk)

Though, it's straightforward enough to just copy the `minio` implementation into here with the license. What's your preference?

**Comment by magik6k** [comment]
go-disk-usage looks good

**Comment by sanderpick** [comment]
Okay, migrated to go-disk-usage.

File: stores/interface.go [hunk 1]
```diff
@@ -38,14 +36,11 @@ type Store interface {
 }
 
 func Stat(path string) (FsStat, error) {
-	var stat syscall.Statfs_t
-	if err := syscall.Statfs(path, &stat); err != nil {
-		return FsStat{}, xerrors.Errorf("statfs: %w", err)
-	}
+	di := du.NewDiskUsage(path)
```
Comment (sanderpick) [inline stores/interface.go:39]: Note, this package doesn't handle the syscall error: https://github.com/ricochet2200/go-disk-usage/blob/master/du/diskusage.go#L16

**Comment by magik6k** [comment]
The same thing is now addressed in https://github.com/filecoin-project/sector-storage/pull/66/files#diff-3edd672f6d345b2738ba957c8bb19960R1

**Comment by magik6k** [comment]
Done in https://github.com/filecoin-project/sector-storage/pull/66

**Comment by sanderpick** [comment]
Sure thing ;)

**Pull request closed without merging**